includes/admin.form.inc

```diff
@@ -94,5 +94,76 @@ function islandora_basic_collection_admin(array $form, array &$form_state) {
       ),
     ),
   );
+
+  // Define the elements that appear on a collection objects display page.
+  // The key's match up with the form elements array keys.
+  $page_content = array(
+    'description' => array(
+      'name' => t("Description"),
+      'description' => t("An objects description field"),
+    ),
+    'collections' => array(
+      'name' => t("In Collection"),
+      'description' => t("Indicates which collections this object belongs to"),
+    ),
+    'wrapper' => array(
+      'name' => t("Fieldset Metadata"),
+      'description' => t("An objects metadata collection set"),
+    ),
+    'islandora_basic_collection_display' => array(
+      'name' => t("Object Content"),
+      'description' => t("Main object page content, such as configured viewers"),
+    ),
+  );
+
+  $form['metadata_display_fieldset']['islandora_basic_collection_metadata_info_table_drag_attributes'] = array(
+    '#theme_wrappers' => array('fieldset'),
+    '#tree' => TRUE,
+    '#title' => t("Page content placement"),
+    '#theme' => 'islandora_basic_collection_metadata_table_drag_components',
+    '#description' => t('Use the table below to determine the rendering order of page and metadata content.'),
+    '#states' => array(
+      'visible' => array(
+        ':input[name="islandora_basic_collection_display_backend"]' => array('value' => 'islandora_solr_query_backend'),
```

Comment:
Should probably exclude the "legacy Sparql" display, instead of including only on the Solr one... If we _do_ want just the Solr one, all this should probably be pulled out to a form alter in the Solr module, [in which the given backend is defined](https://github.com/Islandora/islandora_solr_search/blob/7.x/islandora_solr.module#L470-L482).